enterprise/models.py

```diff
@@ -1528,12 +1528,12 @@ def logo_path(instance, filename):
         filename (str): file to upload
 
     Returns:
-        path: path of image file e.g. enterprise/branding/<model.id>/<model_id>_logo.<ext>.lower()
+        path: path of image file e.g. enterprise/branding/logo_<uuid>.<ext>.lower()
 
     """
     extension = os.path.splitext(filename)[1].lower()
-    instance_id = str(instance.id)
-    fullname = os.path.join("enterprise/branding/", instance_id, instance_id + "_logo" + extension)
+    generated_uuid = str(uuid4())
+    fullname = os.path.join("enterprise/branding/logo_" + generated_uuid + extension)
```

Comment:
imho may be nice to add the `enterprise_customer_uuid` in this path too. it better organizes the files and its a natural partition key for the future, etc.

ex:
`os.path.join("enterprise/branding/" + instance.enterprise_customer.uuid + "/logo_" + generated_uuid + extension)`